From a72a1147b5ddb60e17a2792823ff15252318e11e Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Sun, 29 Oct 2017 12:58:51 +0000
Subject: [PATCH 112/205] mfd: ti_am335x_tscadc: Remove redundant assignment
 to node

Node is being initialized a value that is never read, it is
being written over a few statements into the function with
the return value from call to of_get_child_by_name. Hence
this initialization can be removed. Cleans up clang warning:

drivers/mfd/ti_am335x_tscadc.c:127:22: warning: Value stored
to 'node' during its initialization is never read

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Lee Jones <lee.jones@linaro.org>
(cherry picked from commit e368866ea7a88f5ea16544c4e037b22d993dc3df)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/mfd/ti_am335x_tscadc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mfd/ti_am335x_tscadc.c b/drivers/mfd/ti_am335x_tscadc.c
index 0f3fab4..3cd958a 100644
--- a/drivers/mfd/ti_am335x_tscadc.c
+++ b/drivers/mfd/ti_am335x_tscadc.c
@@ -124,7 +124,7 @@ static	int ti_tscadc_probe(struct platform_device *pdev)
 	struct ti_tscadc_dev	*tscadc;
 	struct resource		*res;
 	struct clk		*clk;
-	struct device_node	*node = pdev->dev.of_node;
+	struct device_node	*node;
 	struct mfd_cell		*cell;
 	struct property         *prop;
 	const __be32            *cur;
-- 
1.7.9.5

